packages/wonder-blocks-form/src/components/field-heading.js

```diff
@@ -21,12 +22,12 @@ type Props = {|
     /**
      * The title for the label element.
      */
-    label: string | React.Element<Typography>,
+    label: string | React.Element<Typography | I18nInlineMarkup>,
```

Comment:
I bet there are a *ton* of places in Wonder Blocks that don't handle this right now, not just field headings - should we be updating the other ones too?

Reply:
Probably.  I can expand the scope of this PR.

Reply:
I've converted this to a draft and will resubmit once I've updated all the other places.